docs/documentation/release_notes/topics/25_0_0.adoc

```diff
@@ -33,6 +38,13 @@ The following methods for setting custom cookies have been removed:
 * `HttpCookie` - replaced by `NewCookie.Builder`
 * `HttpResponse.setCookieIfAbsent(HttpCookie cookie)` - replaced by `HttpResponse.setCookieIfAbsent(NewCookie cookie)`
 
+= Addressed 'You are already logged in' for expired authentication sessions
+
+In Keycloak 23 release, there was improvement added for the case when user authenticated in parallel in multiple browser tabs. However it did not handled well the case when authentication session
+expired. Now for the case when user is already logged-in in one browser tab and authentication session expired in other browser tabs, {project_name} is able to redirect back to the client
+application with some OIDC/SAML error, so client application can immediately retry authentication, which should usually automatically login the application because of the SSO session. More
```

Comment:
```suggestion
application with a OIDC/SAML error, so the client application can immediately retry authentication, which should usually automatically log in the application because of the SSO session. More
```